check-ecs-exec.sh

```diff
@@ -584,24 +584,29 @@ else
   ### CloudWatch Logs
   if [[ ! "x${cloudWatchLogGroupName}" = "xnull" ]]; then
     printf "${COLOR_DEFAULT}     -----\n"
-    # For Resource "*"
+    printf "${COLOR_DEFAULT}     For Resource "*"\n"
+    printf "${COLOR_DEFAULT}     -----\n"
     logsDescribeLogGroup="logs:DescribeLogGroups"
     logsDescribeLogGroupEvalResult=$(${AWS_CLI_BIN} iam simulate-principal-policy \
       --policy-source-arn "${taskRoleArn}" \
       --action-names "${logsDescribeLogGroup}" \
       --output json \
       | jq -r ".EvaluationResults[0].EvalDecision")
     showEvalResult "${logsDescribeLogGroupEvalResult}" "${logsDescribeLogGroup}"
-    # For Resource "${cloudWatchLogGroupName}"
+    printf "${COLOR_DEFAULT}     -----\n"
+    printf "${COLOR_DEFAULT}     For Resource "${cloudWatchLogGroupName}"\n"
+    printf "${COLOR_DEFAULT}     -----\n"
     cwlogGroupArn="arn:aws:logs:${AWS_REGION}:${ACCOUNT_ID}:log-group:${cloudWatchLogGroupName}:*"
     logsCreateLogStream="logs:CreateLogStream"
     logsDescribeLogStreams="logs:DescribeLogStreams"
     logsPutLogEvents="logs:PutLogEvents"
     logsEvalResultsJson=$(${AWS_CLI_BIN} iam simulate-principal-policy \
       --policy-source-arn "${taskRoleArn}" \
-      --action-names "${logsCreateLogStream}" "${logsDescribeLogStreams}" "${logsPutLogEvents}" \
+      --action-names "${logsCreateLogStream}" "${logsDescribeLogStreams}" "${logsPutLogEvents}" "${logsDescribeLogGroup}"\
       --resource-arns "${cwlogGroupArn}" \
       --output json)
+    logsDescribeLogGroupResult=$(readEvalDecision "${logsEvalResultsJson}" "${logsDescribeLogGroup}")
+    showEvalResult "${logsDescribeLogGroupResult}" "${logsDescribeLogGroup}"
```

Comment:
Will we get correct result here since I think it validates against _none_ log group ARN `"arn:aws:logs:${AWS_REGION}:${ACCOUNT_ID}:log-group:${cloudWatchLogGroupName}:*"`?